rxfirebase/src/main/java/com/ezhome/rxfirebase2/database/RxFirebaseDatabase.java

```diff
@@ -101,7 +101,7 @@ public RxFirebaseDatabase observeOn(Scheduler observeOnScheduler) {
         });
         ref.setValue(object);
       }
-    }).compose(this.<String>applyScheduler());
+    }).onBackpressureBuffer().compose(this.<String>applyScheduler());
```

Comment:
I was thinking to make it as RxJava2 where you define the type of backpressure instead of keeping it internally.